torch/CMakeLists.txt

```diff
@@ -230,6 +230,14 @@ else()
     # warnings, see https://bugs.llvm.org/show_bug.cgi?id=21629
     -Wno-missing-braces
     )
+
+  if(NOT APPLE)
+    target_compile_options(torch PRIVATE
+      # Considered to be flaky.  See the discussion at
+      # https://github.com/pytorch/pytorch/pull/9608
+      -Wno-maybe-uninitialized)
```

Comment:
Why is this condition on NOT APPLE?

Reply:
the compiler on the macos CI machines doesn't recognize the existence of this option